chemie/merch/templates/merch/list_all.html

```diff
@@ -0,0 +1,107 @@
+{% extends "chemie/base.html" %}
+{% load material_form %}
+{% load static %}
+
+{% block nav_header %}
+  Merch
+{% endblock nav_header %}
+
+{% block nav_sub_header %}
+  Selges av styret under kontortid!
+{% endblock nav_sub_header %}
+
+{% block content %}
+
```

Comment:
Add check if there exists any merch. If not, display a message of some sort suggesting to create some merch.
Also, remove pagination if there is only one page